spec/models/post_spec.rb

```diff
@@ -0,0 +1 @@
+require 'rails_helper'
```

Comment:
- [ ] Kindly include unit test for your post

Reply:
Thank you ,its done